### DISCUSSION TITLE
SSRIs + Psychedelics (LSD, shrooms, mescaline, DMT, 5-meo-dmt)

### DISCUSSION THREAD

**Discussion by zelixir25** [TripSit/drugs#87] [Drug Interactions]
just providing sources for these combinations

Barbut Siva, J., Barba, T., Kettner, H., Kuc, J., Nutt, D. J., Carhart-Harris, R., & Erritzoe, D. (2024). Interactions between classic psychedelics and serotonergic antidepressants: Effects on the acute psychedelic subjective experience, well-being and depressive symptoms from a prospective survey study. Journal of Psychopharmacology, 38(2), 145–155. https://doi.org/10.1177/02698811231224217

Bonson, K. R., Buckholtz, J. W., & Murphy, D. L. (1996). Chronic administration of serotonergic antidepressants attenuates the subjective effects of LSD in humans. Neuropsychopharmacology, 14(6), 425–436. https://doi.org/10.1016/0893-133X(95)00145-4

Bonson, K. R., & Murphy, D. L. (1996). Alterations in responses to LSD in humans associated with chronic administration of tricyclic antidepressants, monoamine oxidase inhibitors or lithium. Behavioural Brain Research, 73(1–2), 229–233. https://doi.org/10.1016/0166-4328(96)00102-7

Malcolm, B., & Thomas, K. (2022). Serotonin toxicity of serotonergic psychedelics. Psychopharmacology, 239(6), 1881–1891. https://doi.org/10.1007/s00213-021-05876-x

**Comment by utaninja** [comment]
Under the psychedelic umbrella for the chart-
LSD, Shrooms, Mescaline, 2C-x, 2C-T-x, DMT

Barbut Siva, J., Barba, T., Kettner, H., Kuc, J., Nutt, D. J., Carhart-Harris, R., & Erritzoe, D. (2024). Interactions between classic psychedelics and serotonergic antidepressants: Effects on the acute psychedelic subjective experience, well-being and depressive symptoms from a prospective survey study. Journal of Psychopharmacology, 38(2), 145–155. https://doi.org/10.1177/02698811231224217
This is a fantastic study. I just want to note how great this is. :)

Noting two studies are LSD specific - Bonson, K. R., Buckholtz, J. W., & Murphy, D. L. (1996). Chronic administration of serotonergic antidepressants attenuates the subjective effects of LSD in humans. Neuropsychopharmacology, 14(6), 425–436. https://doi.org/10.1016/0893-133X(95)00145-4
Bonson, K. R., & Murphy, D. L. (1996). Alterations in responses to LSD in humans associated with chronic administration of tricyclic antidepressants, monoamine oxidase inhibitors or lithium. Behavioural Brain Research, 73(1–2), 229–233. https://doi.org/10.1016/0166-4328(96)00102-7

Before pushing this- there may be some concern for psychedelics that have MAOI activity.

5-MeO-DMT - MAOI:
"Harmaline, 5-MeO-DMT and bufotenine act agonistically on serotonergic systems and may result in hyperserotonergic effects or serotonin toxicity."
Shen HW, Jiang XL, Winter JC, Yu AM. Psychedelic 5-methoxy-N,N-dimethyltryptamine: metabolism, pharmacokinetics, drug interactions, and pharmacological actions. Curr Drug Metab. 2010 Oct;11(8):659-66. doi: 10.2174/138920010794233495. PMID: 20942780; PMCID: PMC3028383.

**Reply by utaninja** [reply]
Revised and reposted relevant MAOI discussion - https://github.com/TripSit/drugs/discussions/314

**Comment by utaninja** [comment]
Pushed to https://github.com/TripSit/drugs/issues/313
